content/applications/productivity/to_do.rst

```diff
@@ -0,0 +1,148 @@
+
+=====
+To-do
+=====
+
+The To-Do app assists users in efficiently organizing and managing their work. It enables seamless
+collaboration and streamlined workflows across different aspects of the business, offering features
+such as creating and assigning tasks, setting deadlines, creating memos and to-do lists, monitoring
+progress, and much more.
+
+Configuration
+=============
+
+If To-Do is not yet installed on your database, go to the :guilabel:`Apps`, search for it, and click
+:guilabel:`Install`.
+
+.. image:: to_do/install-to-do.png
+   :align: center
+   :alt: Install To-do app
+
+.. _to-do/personal-stages:
+
+Personal stages
+===============
+
+Your assigned to-dos are displayed on your dashboard in different by-default stages:
```

Comment:
Your assigned to-dos are displayed on your dashboard in different default stages: